README

```diff
@@ -278,6 +278,15 @@ Features
     - oracle
       Query data from an Oracle database.
 
+    - ovs_events
+      The plugin monitors the link status of OVS connected interfaces,
+      dispatches the values to collectd and send the notification whenever
+      the link state change occurs in OVS DB. It requires YAJL library to be
+      installed.
+      Detailed instructions for installing and setting up Open vSwitch, see
+      OVS documentation.
+      <http://openvswitch.org/support/dist-docs/INSTALL.md.html>
```

Comment:
Heh, that link gives a 404. Should this point to https://github.com/openvswitch/ovs/blob/master/INSTALL.rst instead?